chrome_extension/vendor/mooltipass/device.js

```diff
@@ -709,6 +709,7 @@ mooltipass.device.messageListener = function(message, sender, sendResponse) {
         if (!message.deviceStatus.connected)
         {
                 mooltipass.device.retrieveCredentialsQueue = [];            
+                mooltipass.backend.disableNonUnlockedNotifications = false;
```

Comment:
If i'm not mistaken, you will then keep having notifications when the device is disconnected no?
Should therefore simply detect when the state transitions between connected and not connected?